### DISCUSSION TITLE
Dandelion output description

### DISCUSSION THREAD

**Discussion by cv55** [tuonglab/dandelion#365] [Q&A]
Hello, thanks for the great tool!
Could you give a brief explanation of what every category means:
1)Orphan V(D)J
2)Extra pair exception

Also I would like to ask one more question about the interpretation of the data. In my case I see that the whole cluster of B-cells fall into the single-pair category. Does this mean that these are all clonal (cancer) cells, and have a unique combination of heavy and light chains?

Thanks a lot!

**Comment by zktuong** [comment]
hi @cv55,

The categories are referring to (for a given cell) how many pairs of BCR there are:

Extra pair = more than 1 pair of IGH+IGK or IGH+IGL e.g. same IGH with different IGK/IGL, different IGH but same IGK/IGL or different IGH and different IGK/IGL
Extra pair-Exception = more than 1 pair of IGH+IGK or IGH+IGL (but flagged as an exception because e.g. IgM/IgD naive B cell, resulting in two IGH contigs)
No_contig = no BCR detected
Orphan VDJ = only IGH detected
Orphan VJ = only IGK or IGL detected
Single Pair = only 1 pair of IGH+IGK or IGH+IGL

So to answer your second question, the single-pair definition doesn't deal with "clonality" just yet - it's just saying that the cell has 1 heavy chain and 1 light chain. You will need to go to the `find_clones` step to address that question.
